tracer/src/Datadog.Trace/Debugger/Instrumentation/AsyncMethodDebuggerState.cs

```diff
@@ -50,14 +52,14 @@ private AsyncMethodDebuggerState()
         internal bool HasArguments { get; set; }
 
         /// <summary>
-        /// Gets or sets a value of the parent (caller) state
+        /// Gets the value of the MethodMetadataInfo that related to the current async method
```

Comment:
```suggestion
        /// Gets the value of the MethodMetadataInfo that is related to the current async method
```
or "relates"

Reply:
Done in f2e39f19694d2e9a44c777ae15e0fa38a3d73526.